debug/log/kubernetes/stream.go

```diff
@@ -22,13 +23,16 @@ type kubeStream struct {
 	stream chan log.Record
 	// the stop chan
 	stop chan bool
+	sync.Mutex
```

Comment:
Place mutex above the thing you're locking, then when someone looks at the struct they know what its for :)